tests/lean/byCasesMetaM.lean.expected.out

```diff
@@ -1,14 +1,4 @@
 [Meta.debug] p q : Prop
     h : p ∧ q
     ⊢ q ∧ q
-[Meta.debug] case inl
-    p q : Prop
-    h : p ∧ q
-    hAux : p
-    ⊢ q ∧ q
-    ------
-    case inr
-    p q : Prop
-    h : p ∧ q
-    hAux : ¬p
-    ⊢ q ∧ q
+[Meta.debug] case inl p q : Prop h : p ∧ q hAux : p ⊢ q ∧ q ------ case inr p q : Prop h : p ∧ q hAux : ¬p ⊢ q ∧ q
```

Comment:
Regression?

Reply:
I don't really know why this PR changed the output here, but none of the spaces here were hard linebreaks (and everything fits on one line).  `ppGoal` uses `Format.line`, and `trace[foo] "bar\n"` *also* uses `Format.line` (because the String→MessageData coercion replaces newlines by `Format.line`).

I've changed `ppGoal` to use hard newlines.